bazel/external/quiche.BUILD

```diff
@@ -4548,35 +4547,68 @@ envoy_cc_library(
     visibility = ["//visibility:public"],
     deps = [
         ":quiche_common_platform_bug_tracker",
-        ":quiche_common_platform_default_quiche_platform_impl_lib",
         ":quiche_common_platform_export",
+        ":quiche_common_platform_flag_utils_impl_lib",
         ":quiche_common_platform_logging",
         ":quiche_common_platform_prefetch",
-        "@envoy//source/common/quic/platform:quiche_common_platform_mem_slice_impl_lib",
+        ":quiche_common_platform_reference_counted_impl_lib",
+        ":quiche_common_platform_testvalue_impl_lib",
+        ":quiche_common_platform_thread_local_impl_lib",
+        ":quiche_common_platform_time_utils_impl_lib",
+        "@envoy//source/common/quic/platform:quiche_mem_slice_impl_lib",
     ],
 )
 
 envoy_quiche_platform_impl_cc_library(
-    name = "quiche_common_platform_default_quiche_platform_impl_export_lib",
+    name = "quiche_common_platform_export_impl_lib",
```

Comment:
Ah, ok. Done. I debated this particular change and thought that the names were clear but quite long so maybe shortening them would be an improvement. But happy to change them back since they're not an improvement. 

Thanks!